spec/swagger_v2/default_api_spec.rb

```diff
@@ -139,4 +139,28 @@ def app
       expect(subject['x-logo']).to eql('http://logo.com/img.png')
     end
   end
+
+  context 'with tags' do
+    def app
+      Class.new(Grape::API) do
+        format :json
+        desc 'This gets something.'
+        get '/something' do
+          { bla: 'something' }
+        end
+        add_swagger_documentation tags: [
+          { name: 'something', description: 'customized description' }
+        ]
```

Comment:
please can you add a second tag, so that it will be clear, that an existing tag can be overwritten and non-existing will be added, thanks